core/src/SYCL/Kokkos_SYCL_Error.hpp

```diff
@@ -0,0 +1,15 @@
+namespace Kokkos {
+namespace Impl {
+/*
+void sycl_internal_error_throw( syclError_t e , const char * name, const char * file = NULL, const int line = 0 );
+
+inline void sycl_internal_safe_call( syclError_t e , const char * name, const char * file = NULL, const int line = 0)
+{
+  if ( syclSuccess != e ) { sycl_internal_error_throw( e , name, file, line ); }
+}
+*/
+}
+}
+#define SYCL_SAFE_CALL( call )  \
+        call
```

Comment:
Did you want to keep this?  This is used in backends where the runtime use error codes.  My understanding is SYCL uses exceptions.